### PR TITLE
chore(asgard): add agard to the project

### PR DESCRIPTION
**Pull request by curzolapierre**

### CODE REVIEW

**Review by Soulou — CHANGES_REQUESTED** [review]
In the dependencies, could you only set Scalingo projects? (I think it's none for this project)

File: .sclng/metadata.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+dependencies = []
+description = "Service monitoring actively containers running on a server"
+destination_server = ["app-hosting-node"]
+flags = ["deployed-by-chef", "tools"]
+languages = ["go"]
+owner = "leo@scalingo.com"
+team = "IST"
+version = "1.0.2"
```
Comment (EtienneM) [inline .sclng/metadata.toml:8]: If there is a version number here, the README should be updated, the section "Release a New Version"

Comment (curzolapierre) [reply .sclng/metadata.toml:8]: This version comes from asgard, cf: https://github.com/Scalingo/asgard-cli/blob/master/metadata/init.go#L70

**Comment by Soulou** [comment]
> I'm not sure to understand what is this field for.

@curzolapierre  For instance `autoscaler-service` NEEDS `alerter-service`

**Review by Soulou — APPROVED** [review]

**Pull request merged**